exercises/gigasecond/example.js

```diff
@@ -1,6 +1,6 @@
 const GIGASECOND_IN_MILIS = 1e9 * 1e3;
```

Comment:
I hate to ask, but while we're here, can we fix the spelling of 'millis'?  Should be `MILLIS`... :)

Reply:
Good eye! I did not even catch that. Will fix.